### PR TITLE
Add Cash App Pay icon

### PR DESCRIPTION
**Pull request by afterpayplugins**
## Why are you adding this icons?
I'm adding/updating this icon(s) because we're Shopify Partners and recently created a new payment app for Cash App Pay

#### Help us identify yourself
- [ ] I'm working/collaborating with the brand directly and they have provided the icons.
- [ x] I'm associated with the brand and I've read all the brand icon’s guidelines.
- [ ] I'm an individual and I've read all the brand icon’s guidelines.

## Checklist to add new icons
- [x ] All icons have a corresponding entry in `db/payment_icons.yml`
- [x] I have followed the icon guidelines detailed in the [CONTRIBUTING.md](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) file
- [x ] I have optimized the icon with [SVGO](https://jakearchibald.github.io/svgomg/)
- [x ] I am confident that all icons are clear and easy to read/understand
- [ ] I have provided a link to the brand icon’s brand guidelines whenever possible.
- [x ] I recognize that if my icon is not approved by the Shopify Partners team it may not receive review nor merger.

## Attach a screenshot of the icon with the black background
By itself:
Screenshot 2022-10-28 at 2 34 33 PM

Next to placeholder:
Screenshot 2022-10-28 at 2 33 58 PM

#### If the icons are intended for use by Shopify, please provide the following info:
Who are you working with at Shopify?
We are Shopify partners and have a payment app that has been reviewed and approved.
https://partners.shopify.com/385371/apps/4293787649/overview

What's the expected date of this change to deploy on Shopify?
ASAP.

### CODE REVIEW

File: db/payment_icons.yml [hunk 1]
```diff
@@ -658,6 +658,10 @@
   name: clearpay
   label: Clearpay
   group: other
+-
+  name: cash_app_pay
```
Comment (lunataoo) [inline db/payment_icons.yml:662]: Please only include alphabetical char in the name, like "cashapppay"

**Review by lunataoo — DISMISSED** [review]
The icon border does not match the standard example, please remove it from the SVG and update the name. TY!
SVG Editor:Viewer Online 2022-11-14 16-30-57

**Comment by lunataoo** [comment]
Please use the provided [template shape and border here](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#template).
SVG Editor:Viewer Online 🔊 2022-11-18 17-09-32
Monosnap 2022-11-18 17-12-23

**Comment by abbadata** [comment]
> Please use the provided [template shape and border here](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#template). SVG Editor:Viewer Online 🔊 2022-11-18 17-09-32 Monosnap 2022-11-18 17-12-23

Thanks for the clarification. We've updated the icon as well as the icon name. Thanks!

**Comment by abbadata** [comment]
@lunataoo I made the requested changes. Please let me know if there's anything else I need to do. Thanks!

**Review by Jacquesattaque — APPROVED** [review]

**Pull request merged**